FileShareAdminClient/FileShareApiAdminClient.cs

```diff
@@ -205,5 +206,19 @@ public async Task RollBackBatchAsync(IBatchHandle batchHandle)
                 response.EnsureSuccessStatusCode();
             }
         }
+
+        public async Task<HttpResponseMessage> ReplaceAclAsync(string batchId, Acl acl)
```

Comment:
Please add CancellationToken parameter and replace CancellationToken.None

Reply:
updated as you suggested, Thanks